pandas/core/ops.py

```diff
@@ -125,11 +127,18 @@ def maybe_upcast_for_op(obj):
     Be careful to call this *after* determining the `name` attribute to be
     attached to the result of the arithmetic operation.
     """
-    if type(obj) is datetime.timedelta:
+    if type(obj) is datetime.timedelta or isinstance(obj, Tick):
```

Comment:
can you put these in the same isinstance check

Reply:
Sure.  The cost is that sometimes we'll catch `pd.Timedelta` here, but that's pretty cheap.

Reply:
k, otherwise this lgtm. ping on green.